src/server/server.py

```diff
@@ -765,4 +980,6 @@ def run(self, port=int(SERVER_OPTS['port']), *middleware):
         print('LDAP: %s' % SERVER_OPTS['ldap'])
         print('Admin DB Failover: %s' % SERVER_OPTS['admin_db_failover'])
     APP = MyApplication(URLS, globals())
+    config.debug = False
```

Comment:
Could this be configurable ?

Reply:
Yes, but it's a feature. I'm adding an Issue on my personal board:
https://github.com/nbeguier/cassh/issues/66

Reply:
By the way, did you see https://github.com/leboncoin/cassh/projects :)
A ticket board integrated to Github !